### DISCUSSION TITLE
A bit of RiMusic

### DISCUSSION THREAD

**Discussion by fast4x** [fast4x/RiMusic#85] [Announcements]
I wanted to share with you the news that RiMusic has been downloaded 1000 times in a month.
😉

Thanks to all collaborators.

**Comment by siggi1984** [comment]
> I wanted to share with you the news that RiMusic has been downloaded 1000 times in a month. 😉
> 
> Thanks to all collaborators.

That's great news, congratulations.
I think that this is far from the end.
The app has turned out very well so far and I'm looking forward to seeing how it develops.

**Comment by ikanakova** [comment]
Soon it will also have 1000 stars, especially after it will be available on F-Droid 🙂

**Comment by dainvincible1** [comment]
That's awesome. Congratulations. 
Many more milestone to achieve and mang more achievements to come. 🎉🎉

**Comment by fast4x** [comment]
Hello everyone, RiMusic has been downloaded 2,670 times to date, I find it very promising.

I thank the people who downloaded it and those who want to download it.

Special thanks to all collaborators of RiMusic.

Another special thanks goes to ViMusic and its creator, without which RiMusic would not have existed.

**Comment by fast4x** [comment]
6000 download Is a good target!

Thanks to all

**Comment by fast4x** [comment]
Thanks to all for 11.000 download!

**Reply by ikanakova** [reply]
Congrats!

**Reply by fast4x** [reply]
> Congrats!

Thanks also to you and all the other collaborators

**Comment by fast4x** [comment]
I wanted to share with everyone the news that since the last release it has been downloaded 3000 times.

Overall since September it has been downloaded 31000 times. Wow.

I thank you all for your collaboration.

Thanks thanks thanks

**Reply by SureshTimma** [reply]
We r always with U❤️

**Reply by OrangeZXZ** [reply]
> I wanted to share with everyone the news that since the last release it has been downloaded 3000 times.
> 
> Overall since September it has been downloaded 31000 times. Wow.
> 
> I thank you all for your collaboration.
> 
> Thanks thanks thanks 
> 
> 

i hope this doesn't give you too mutch work lol congrats !

**Reply by ikanakova** [reply]
Congrats! 🎉

**Comment by SureshTimma** [comment]
What is invisible service mean?

What is the use of cache?

If you know please tell me....

**Reply by twistios** [reply]
The cache stores songs so that you can listen to them while offline.
